src/generator/index.tsx

```diff
@@ -66,25 +67,17 @@ export default async function render({ messages, channel, callbacks, ...options
         <title>{channel.isDMBased() ? 'Direct Messages' : channel.name}</title>
 
         {/* message reference handler */}
-        <script
-          dangerouslySetInnerHTML={{
-            __html: scrollToMessage,
-          }}
-        />
+        <script>{scrollToMessage}</script>
```

Comment:
Does this work as intended? I remember having to use `dangerouslySetInnerHTML` otherwise React would escape the script contents.